charts/fluentd/README.md

```diff
@@ -1,9 +1,227 @@
+```
+
+To install a release named `fluentd`, run:
+
+```sh
+helm install fluentd fluent/fluentd
+```
 
 ## Chart Values
 
 ```sh
 helm show values fluent/fluentd
 ```
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| nameOverride | string | `""` |  |
+| fullnameOverride | string | `""` |  |
+| kind | string | `"DaemonSet"` | The kind of kubernetes controller to use "DaemonSet" or "Deployement" |
+| replicaCount | int | `1` | Only for kind: Deployment |
+| image.pullPolicy | string | `"IfNotPresent"` |  |
+| image.repository | string | `"fluent/fluentd-kubernetes-daemonset"` |  |
+| image.tag | string | `"v1.11.5-debian-elasticsearch7-1.0"` |  |
+| imagePullSecrets | list | `[]` | Optional array of imagePullSecrets containing private registry credentials |  |
+| serviceAccount.create | bool | `true` |  |
+| serviceAccount.annotations | object | `{}` |  |
+| serviceAccount.name | string | `null` |  |
+| rbac.create | bool | `true` |  |
+| podSecurityPolicy.enabled | bool | `true` |  |
+| podSecurityPolicy.annotations | object | `{}` |  |
+| podSecurityContext | object | `{}` |  |
+| securityContext | object | `{}` |  |
+| resources | object | `{}` |  |
+| priorityClassName | string | `""` |  |
+| nodeSelector | object | `{}` |  |
+| tolerations | list | `[]` |  |
+| affinity | object | `{}` |  |
+| podAnnotations | object | `{}` |  |
+| env[0].name | string | `"FLUENTD_CONF"` |  |
+| env[0].value | string | `"/etc/fluent/fluent.conf"` |  |
+| envFrom | list | `[]` |  |
+| volumes | list | [default-volumes](#default-volumes) | The external volumes to use |
+| volumeMounts | list | [default-volumeMounts](#default-volumeMounts) | The volume to mount with the pods |
+| service.type | string | `"ClusterIP"` |  |
+| service.annotations | object | `{}` |  |
+| service.ports | list | `[]` |  |
+| metrics.serviceMonitor.enabled | bool | `false` |  |
+| metrics.serviceMonitor.additionalLabels | object | `{release: prometheus-operator}` |  |
+| metrics.serviceMonitor.namespace | string | `""` |  |
+| metrics.serviceMonitor.namespaceSelector | object | `{}` |  |
+| metrics.serviceMonitor.scrapeInterval | string | `null` |  |
+| metrics.serviceMonitor.scrapeTimeout | string | `null` |  |
+| metrics.serviceMonitor.honorLabels | bool | `true` |  |
+| metrics.prometheusRule.enabled | bool | `false` |  |
+| metrics.prometheusRule.additionalLabels | object | `{}` |  |
+| metrics.prometheusRule.namespace | string | `""` |  |
+| metrics.prometheusRule.rules | list | `[]` |  |
+| dashboards.enabled | bool | `true` |  |
+| dashboards.additionalLabels | object | `{grafana_dashboard: "1"}` |  |
+| dashboards.namespace | string | `""` |  |
+| plugins | list | `[]` | Fluentd list of plugins to install |
+| configMapConfigs | list | `["fluentd-prometheus-conf"]` | List of fluentd.conf type configMaps to mount |
+| fileConfigs | object | [default-volumes](#default-fluentdConfig) | Configuration files to be read by the fluentd engine |
```

Comment:
this table can be removed, it will be tough to maintain and `helm show values` suffices today